barretenberg/cpp/src/barretenberg/stdlib/primitives/group/cycle_scalar.cpp

```diff
@@ -17,37 +17,6 @@ cycle_scalar<Builder>::cycle_scalar(const field_t& _lo, const field_t& _hi)
     , hi(_hi)
 {}
 
-template <typename Builder> cycle_scalar<Builder>::cycle_scalar(const field_t& in)
```

Comment:
previously only used in the transcript `split_challenge` which now uses an analogous `field_t` method (as it always should have).